po/nb_NO.po

```diff
@@ -3,457 +3,452 @@ msgstr ""
 "Project-Id-Version: clapper\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-09-14 16:35+0200\n"
-"PO-Revision-Date: 2021-09-14 15:25\n"
-"Last-Translator: \n"
-"Language-Team: Norwegian\n"
-"Language: no_NO\n"
+"PO-Revision-Date: 2021-10-01 23:50\n"
+"Last-Translator: Allan Nordhøy <epost@anotheragency.no>\n"
+"Language-Team: Liberium\n"
+"Language: nb_NO\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Crowdin-Project: clapper\n"
-"X-Crowdin-Project-ID: 473374\n"
-"X-Crowdin-Language: no\n"
-"X-Crowdin-File: /master/po/com.github.rafostar.Clapper.pot\n"
-"X-Crowdin-File-ID: 31\n"
 
 #: ui/clapper.ui:6
 msgid "Open Files..."
-msgstr ""
+msgstr "Åpne filer …"
```

Comment:
Are space and single three dots char preferred over simple 3x dot char?

Reply:
The space is just the rules in nb_NO. Ellipsis is correct and triple-dot is not.